class7-week3/actions/clear.js

```diff
@@ -0,0 +1,19 @@
+const Todo = require('../models/todo')
+
+module.exports = function clear (request, response) {
+
+
+    Todo.clear (error => {

```

Comment:
Conceptually, yes, but `clear is not a function`. If you want to do this you need to implement it in `models/todo` (a good idea)